storagenode/storagenodedb/reputation.go

```diff
@@ -88,11 +96,40 @@ func (db *reputationDB) Get(ctx context.Context, satelliteID storj.NodeID) (_ *r
 		&stats.Audit.Alpha,
 		&stats.Audit.Beta,
 		&stats.Audit.Score,
+		&stats.Disqualified,
 		&stats.UpdatedAt,
 	)
+
+	return &stats, ErrReputation.Wrap(err)
+}
+
+// All retrieves all stats from DB
+func (db *reputationDB) All(ctx context.Context) (_ []reputation.Stats, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	rows, err := db.QueryContext(ctx, `SELECT * FROM reputation`)
```

Comment:
I would feel more comfortable with something like `SELECT satellite_id, uptime_success_count, ... FROM reputation` rather than `SELECT * FROM reputation` because what if the order that you put arguments into `rows.Scan` below does not match up with the column order returned by using `*`?

I think either you should have a test to verify that this function works as expected, or update the query to be specific about the columns and order of columns returned.

Reply:
Yeah, agree, good point